chessboard/index.js

```diff
@@ -0,0 +1,45 @@
+
+
+const container = document.getElementById("chess-container");
+
+const createBoard = () => {
+    for (let i = 0; i < 8; i++) {
+        container.insertAdjacentHTML("beforeend", `<div>
+        <span id=${i * 8 + 0}></span>
+        <span id=${i * 8 + 1}></span>
+        <span id=${i * 8 + 2}></span>
+        <span id=${i * 8 + 3}></span>
+        <span id=${i * 8 + 4}></span>
+        <span id=${i * 8 + 5}></span>
+        <span id=${i * 8 + 6}></span>
+        <span id=${i * 8 + 7}></span>
+        </div>`);
+    }
+}
+
+createBoard();
+
+container.addEventListener("mouseover", (event) => {
+    const hoveredSpanId = Number(event.target.id);
+    const row = Math.floor(hoveredSpanId / 8);
+    const col = hoveredSpanId % 8;
+    
+    for (i = 0; i < 8; i++) {
+        const rowDif = Math.abs(row - i);
+        // right side diagnol elements on each row, except hovered element row      
+        if (i !== row && i * 8 + (col + rowDif) < (i + 1) * 8) {
+            document.getElementById(`${i * 8 + (col + rowDif)}`).className = "highlight"
```

Comment:
Better use classList.add() method instead of directly assigning the className here

`            document.getElementById(`${i * 8 + (col + rowDif)}`).classList.add('highlight')
`